README.rst

```diff
@@ -77,6 +77,7 @@ configuration each task uses.
             "app": "appname",
             "aws": {
                 "region": "us-west-2",
+                "profile_name": "my-aws-profile",  # a profile from .aws/credentials
```

Comment:
Is the idea that since not all tasks require this method, this would be set temporarily in your local checkout, run the tasks, and then unset it? If not, I wonder if everyone is using the same `AWS_PROFILE` name or not.

Reply:
Good question. I was going on the assumption that we standardize the AWS_PROFILE that each developer uses for the project. The [docs in jade-truffle](https://github.com/caktus/jade-truffle/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/docs/deploy.md#1-configure-aws-named-profile) now include a section for setting up a profile named after the project, and that's what we're doing in philly-hip. But maybe I should make that more explicit in the docs or here?

Reply:
Ah right, and this is just the playbook task, not deploy. I suppose if you wanted an optional profile, you could write a project-level task to just set it ad-hoc, like `inv staging temp-creds playbook`.

So I think making it more explicit in the docs sounds good 👍